src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt

```diff
@@ -57,12 +50,16 @@ class DiploidPathFinding: CliktCommand(help = "Impute best diploid path using re
         .required()
         .validate { require(File(it).isDirectory) {"$it is not a valid directory."} }
 
+    val pathType by option(help = "The type of path to find. Must be lower case. Haploid infers a single path through the graph. Diploid infers a pair of paths.")
```

Comment:
should this have a default?

Reply:
I prefer to force the user to include pathType in the command so that it is obvious what type of path is being fit.

Reply:
you may want to change the help to be the following so it is clear what the two inputs are
`The type of path to find. Must be lower case. 'haploid' infers a single path through the graph. 'diploid' infers a pair of paths.`

Reply:
Good point. I will make the change.